pkgs/by-name/la/laravel/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  php,
+}:
+php.buildComposerProject (finalAttrs: {
+  pname = "laravel";
+  version = "5.8.1";
+
+  src = fetchFromGitHub {
+    owner = "laravel";
+    repo = "installer";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-LQABZnmKgJ8qkymmSjhjc+x1qZ/tFqFyQbfeGZECxok=";
+  };
+
+  composerLock = ./composer.lock;
+  vendorHash = "sha256-f18N2qNCUFetCaHaC4X6Benq70x21SVQ3YSs8kovK1g=";
+
+  meta = with lib; {
```

Comment:
`with lib;` is an anti-pattern, see here https://github.com/NixOS/nixpkgs/issues/208242.

Reply:
maintainers = with lib.maintainers; [ <someone> ]: is fine?

Reply:
Yeah that's fine.